packages/php-wasm/universal/src/lib/php.ts

```diff
@@ -681,6 +681,9 @@ export class PHP implements Disposable {
 					requestHeaders,
 					port
 				);
+				if (request.relativeUri) {
+					this.#setServerGlobalEntry('PHP_SELF', request.relativeUri);
```

Comment:
This is also set by PHPRequestHandler. I know this class is sometimes used outside of PHPRequestHandler, but when using PHPRequestHandler, does this line overwrite the PHP_SELF set by PHPRequestHandler?